week6/04-grazhdanskaia-oborona/task.py

```diff
@@ -0,0 +1,49 @@
+def main():
+    n = int(input())
+    village_list = list(input().split())
+    m = int(input())
+    bomb_list = list(input().split())
+    for i in range(n):
+        pos = village_list[i]
+        village_list[i] = (int(pos), i + 1)
+    for i in range(m):
+        pos = bomb_list[i]
+        bomb_list[i] = (int(pos), i + 1)
+    village_list.sort(key=lambda i: i[0])
+    bomb_list.sort(key=lambda i: i[0])
```

Comment:
confusing - i is usually conter in loop and had this meaning in prev code. use el